compute_endpoint/globus_compute_endpoint/cli.py

```diff
@@ -413,7 +421,11 @@ def configure_endpoint(
             "high-assurance(HA) endpoints require both a HA policy and "
             "a HA subscription id"
         )
+    elif auth_policy_mfa_required and not create_policy:
+        raise ClickException("MFA can only be specified when creating a policy")
     elif create_policy:
+        if auth_policy_mfa_required and not high_assurance:
+            raise ClickException("MFA can only be enabled for high-assurance policies")
```

Comment:
- Grammar nitpick: `s/can/may/g`
- Should we capitalize High-Assurance?

Reply:
I changed all high-assurance texts in this section to be 'High-Assurance(HA)'.

I think I got the original spelling from Globus's previous HA pages, but it's a mix.  As https://docs.globus.org/guides/overviews/high-assurance/ has more capitalized rather than not, I changed everything to the cap version.

Reply:
LGTM.